### ISSUE TITLE
detect speed

### ISSUE THREAD

**Issue by KeyKy** [Seanlinx/mtcnn#15]
I found the speed of detection is about 1.0s/image (some images are 0.3s/images). Is it right?

**Comment by Seanlinx** [comment]
The speed is about 50ms per image for VGA images(30ms-40ms per image for fddb images) on titan x.

**Comment by KeyKy** [comment]
@Seanlinx which mode do you use? slide windows or fcn.

**Comment by Seanlinx** [comment]
fcn @KeyKy

**Comment by KeyKy** [comment]
@Seanlinx I use slide windows. Is it slower than fcn?

**Comment by KeyKy** [comment]
When i use fcn i get this error Check failed: x.ctx() == default_ctx All arguments must be in global context unless group2ctx is specified

**Comment by KeyKy** [comment]
here is some time cost using fcn and the model you provided.
time cost 0.744  pnet 0.733  rnet 0.009  onet 0.003
time cost 0.833  pnet 0.820  rnet 0.009  onet 0.004
time cost 0.106  pnet 0.096  rnet 0.007  onet 0.002
time cost 0.179  pnet 0.170  rnet 0.007  onet 0.003
time cost 0.039  pnet 0.028  rnet 0.009  onet 0.003
time cost 0.048  pnet 0.029  rnet 0.012  onet 0.007
time cost 0.058  pnet 0.042  rnet 0.009  onet 0.006

**Comment by Seanlinx** [comment]
Yes. And it'll be faster if you set MXNET_ENGINE_TYPE to NaiveEngine

**Comment by tuner24** [comment]
hi,  i run the demo.py on 1080 ti, fcn cost 0.48s, slide window cost 0.18s, i don't know why.
should i test multi images, or test one image multi times then i can get the 50ms performance? @KeyKy @Seanlinx

**Comment by tzhang2014** [comment]
@Seanlinx i use the demo to test a 1080P video , run time is 50ms, is it normal ?my PC is i7-6700 ,GPU is GTX 1070, thx
